Examples/Algorithms/TrackFitting/include/ActsExamples/TrackFitting/TrackFittingAlgorithm.hpp

```diff
@@ -31,7 +31,8 @@ class TrackFittingAlgorithm final : public BareAlgorithm {
   /// Track fitter function that takes input measurements, initial trackstate
   /// and fitter options and returns some track-fitter-specific result.
   using TrackFitterOptions =
-      Acts::KalmanFitterOptions<MeasurementCalibrator, Acts::VoidOutlierFinder>;
+      Acts::KalmanFitterOptions<MeasurementCalibrator, Acts::VoidOutlierFinder,
+                                Acts::VoidReverseFilteringLogic>;
```

Comment:
In principle, this could be defaulted in the constructor of the options, right?

Reply:
I think in principle we could but do we want to ? I don't know if there was a reason for not doing it in the past ?